README.md

```diff
@@ -359,6 +359,7 @@ _See also: [IT Asset Management]([Ralph](#it-asset-management))_
 * [OpenDS](https://opends.java.net/) - Another directory server written in Java.
 * [OpenLDAP](http://www.OpenLDAP.org/) - Developed by the OpenLDAP Project.
 * [FreeIPA](http://www.freeipa.org/) - Based on 389-DS. Includes Kerberos, DNS, as well as host based access control.
+* [lldap](https://github.com/nitnelave/lldap) - Light (simplified) LDAP implementation written in Rust and a web interface with Node.js ([Source Code](https://github.com/nitnelave/lldap)) `GPL-3.0` `Rust`/`JavaScript`.
```

Comment:
Just a comment from the author: There's no JS (unless you count the 5 lines to bootstrap the WASM frontend). The web interface is _not_ in Node.js but in Rust compiled to WALM (with the Yew framework, if it's relevant).

In the one-line blurb, I would maybe talk more about the main goals: easy to deploy and to use (but less featureful than other LDAP servers).